DOKS-CI-CD/README.md

```diff
@@ -0,0 +1,1340 @@
+    - URL: https://github.com/knative/net-certmanager/releases/download/knative-v1.4.0/release.yaml
+```
+
+You don't have to apply the above manifest by hand, because everything is handled via [Kustomize](https://kustomize.io) in this repository (including enabling the auto TLS feature for Knative Services).
+
+What Kustomize does in this blueprint (via the [kustomization](assets/manifests/knative-serving/kustomization.yaml) manifest), is to take the original [KnativeServing configuration](https://github.com/digitalocean/marketplace-kubernetes/blob/master/stacks/knative/assets/manifests/knative-serving.yaml) from the DigitalOcean marketplace GitHub repository, and apply a set of patches for each feature that is needed. This way, you don't have to touch the original file, or keep a copy of it somewhere with your own changes. Kustomize empowers you to use the original file as a base, and apply a set of patches on the fly (very powerful).
+
+Each patch file and purpose is listed below:
+
+- [net-certmanager-install](assets/manifests/knative-serving/patches/net-certmanager-install.yaml) - installs `net-certmananger` component via Knative Operator.
+- [certmanager-config](assets/manifests/knative-serving/patches/certmanager-config.yaml) - configures `Knative Serving` to use the `ClusterIssuer` created earlier to automatically issue TLS certificates from [Let's Encrypt](https://letsencrypt.org).
+- [domain-config](assets/manifests/knative-serving/patches/domain-config.yaml) - configures `Knative Serving` to use your custom `domain` when exposing services.
+- [network-config](assets/manifests/knative-serving/patches/network-config.yaml) - configures `Knative Serving` to enable the `auto TLS` feature for `Knative Services`, via the `auto-tls` flag.
+
+Follow below steps to apply all required changes for the Knative Serving component, via Kustomize:
+
+1. Clone the `container-blueprints` repo (if not already):
+
+    ```shell
+    git clone https://github.com/digitalocean/container-blueprints.git
+    ```
+
+2. Change directory to your local copy:
+
+    ```shell
+    cd container-blueprints
+    ```
+
+3. Edit [domain-config](assets/manifests/knative-serving/patches/domain-config.yaml) to point to your own domain name (replace the `starter-kit.online` key), using a text editor of your choice (preferably with `YAML` linting support). For example you can use [Visual Studio Code](https://code.visualstudio.com):
+
+    ```shell
+    code DOKS-CI-CD/assets/manifests/knative-serving/patches/domain-config.yaml
+    ```
+
+4. Apply `kustomizations` using kubectl (since `1.14`, kubectl also supports the management of Kubernetes objects using a kustomization file, via the `-k` flag):
+
+    ```shell
+    kubectl apply -k DOKS-CI-CD/assets/manifests/knative-serving
+    ```
+
+Finally, you can deploy the sample `hello-world` knative service provided in the DigitalOcean marketplace repo to test the whole setup:
+
+```shell
+kubectl apply \
+  -n doks-ci-cd \
```

Comment:
This namespace "doks-ci-cd" is created in Step 7 now. It's probably best to move it in Step 5 before this command needs to be run.

Reply:
Thanks. Fixed in latest commit.